types/faker/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'faker'
```

Comment:
you would need to exclude this folder from linting

Reply:
@rostislav-simonik-plc looks like it is excluded already https://github.com/plentific/eslint-plugin-plentific/blob/main/tsconfig.json#L27 - am I checking the right file?

Reply:
@asligulsarpel that's exclusion for typescript not eslint, you need introduce ignorePatterns in config or .eslintignore file. https://eslint.org/docs/latest/user-guide/configuring/ignoring-code